claim-db-worker/components/Navbar.tsx

```
@@ -0,0 +1,126 @@
+                </div>
+              ) : !isLoading ? (
+                <span className="text-muted text-sm font-bold leading-none">
+                  Database Claimed
+                </span>
+              ) : null}
+            </div>
+            {!isLoading && timeRemaining && timeRemaining > 0 && (
+              <div
+                className="w-40 px-4 py-2 bg-teal-600 hidden lg:flex rounded-md justify-center items-center cursor-pointer hover:bg-teal-700 transition-colors"
+                onClick={handleClaimDatabase}
+              >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)